ui/StatusQ/src/StatusQ/Components/StatusEmojiAndColorComboBox.qml

```diff
@@ -24,7 +24,7 @@ import StatusQ.Core.Theme 0.1
    \qml
             StatusEmojiAndColorComboBox {
                 Layout.preferredWidth: 300
-                model: WalletAccountsModel {}
+                model: SFPMWalletAccountsModel {}
```

Comment:
Does this then use the new component from storybook?

Reply:
Wrong replacement! :) I'll update it